src/App.jsx

```diff
@@ -14,70 +16,60 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [value, setValue] = useState('Jam');
```

Comment:
value it is soo general name, it's better to set specific name, for instance: **selectedGood**